docs/en/status-communities/change-a-token-gated-community-to-open.md

```diff
@@ -0,0 +1,43 @@
+---
+id: 637
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Change a token-gated community to open
+
+!!! note ""
+    Currently, you can only set up token-gated and open communities using Status desktop. [:octicons-desktop-download-16: Download Status desktop][status-web-download].
+
+In a [token-gated community][token-based-access-to-communities-and-channels] members should meet specific token requirements to join. As a community owner, you can change an open-gated community into an open one so anyone can join without requiring tokens.
+
+!!! note
+    In addition to token requirements, you can set up [manual approval][about-community-request-approvals] to join your community. Token requirements and manual approval work independently.
```

Comment:
How about 'In addition to token requirements, you can make [manual approval] of new join requests mandatory'? I think this would make the sentence easier to scan for those who have no idea what manual approval is in Status.

Reply:
I applied your suggestion but changed your sentence slightly.